buildkite/test_plans.go

```diff
@@ -0,0 +1,50 @@
+package buildkite
+
+import "fmt"
+
+type TestPlansService struct {
+	client *Client
+}
+
+type Case struct {
+	Path              *string `json:"path,omitempty" yaml:"path,omitempty"`
+	EstimatedDuration *int `json:"estimated_duration,omitempty" yaml:"estimated_duration,omitempty"`
+}
+
+type Tests struct {
+	Format *string `json:"format,omitempty" yaml:"format,omitempty"`
+	Cases  []Case  `json:"cases,omitempty" yaml:"cases,omitempty"`
+}
+
+type Task struct {
+	NodeNumber *int   `json:"node_number,omitempty" yaml:"node_number,omitempty"`
+	Tests      *Tests `json:"tests,omitempty" yaml:"tests,omitempty"`
+
```

Comment:
This empty line might get picked up by a linter.

Reply:
:+1:

Reply:
Fixed by https://github.com/buildkite/go-buildkite/pull/183/commits/77f55d7de1092e71f3c438f2c0b1433beb3f49f0